src/novaprobe.py

```diff
@@ -14,7 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# from pprint import pprint
+#from pprint import pprint
```

Comment:
Actually, remove the unneeded import as it was needed only for debugging purposes.